ReactiveCocoaTests/Swift/FlattenSpec.swift

```diff
@@ -107,6 +107,7 @@ class FlattenSpec: QuickSpec {
 				var observed: Int? = nil
 				outer
 					.flatten(.Latest)
+					.assumeNoErrors()
```

Comment:
I love this change, because it forces the code to explicitly yell "I'm ignoring errors".